charts/zeebe-benchmark/test/golden/c8-core-configmap.golden.yaml

```diff
@@ -90,20 +90,12 @@ data:
               url: "http://benchmark-test-elasticsearch:9200"
               index:
                 prefix: "zeebe-record"
-              retention:
-                enabled: true
-                minimumAge: "1h"
-                policyName: "core-record-retention-policy"
           CamundaExporter:
             className: "io.camunda.exporter.CamundaExporter"
             args:
               connect:
                 type: elasticsearch
                 url: "http://benchmark-test-elasticsearch:9200"
-              retention:
-                enabled: true
-                minimumAge: "1h"
-                policyName: "core-record-retention-policy"
               createSchema: true
```

Comment:
🙃 😢 Looks like something broke again? Because this would mean we are not cleaning up data